agc-coffea-2024.py

```diff
@@ -42,21 +42,31 @@
 import numpy as np
 import uproot
 from dask.distributed import Client
+import cloudpickle
+import matplotlib.pyplot as plt
 
 from coffea.nanoevents import NanoEventsFactory, NanoAODSchema
 from coffea.analysis_tools import PackedSelection
 from coffea import dataset_tools
+import correctionlib
 
 import warnings
 
 import utils
+from utils.systematics import rand_gauss
 utils.plotting.set_style()
 
 warnings.filterwarnings("ignore")
 NanoAODSchema.warn_missing_crossrefs = False # silences warnings about branches we will not use here
 
-
-client = Client("tls://localhost:8786")
+import getpass
+whoami_output = getpass.getuser()
+if whoami_output == "cms-jovyan":
```

Comment:
This will only work on some instances, @oshadura do you have an idea for how to generalize that?

Reply:
Oh but I think it can also be removed. It was convenient for me because I was running tests both on coffea-casa and elsewhere, but one can just add it if needed.

Reply:
I cant imagine a simple way to detect where you are on local laptop or facility: I think it will more sence to add "profiles", where you say if you are using AF="coffea-casa" or AF="local"

Reply:
I just removed it, it will be used on coffea-casa anyways so there was no point in that.